akka-operator/values.yaml

```diff
@@ -38,3 +42,21 @@ resources:
 nodeSelector: {}
 tolerations: []
 affinity: {}
+# valid values:
+# - false (default)
+# - aws
+# - gcp
+#
+# use `--set` in `helm install` to set from command line
+#
+# if (gcp && imageRepository == false) {
+#   use default gcpRepo
+# } else if (aws && imageRepository == false) {
+#   use default awsRepo
+# } else {
+#   use imageRepository (e.g. will fail if false)
+# }
+#
+# default aws and gcp repos will be present in the script for `helm install`
+provider:
+  name: false
```

Comment:
remember to update aws-install.adoc in the platform guide when this is merged

Reply:
sure .. will do ..

Reply:
thinking about reverting this actually :-)